### DISCUSSION TITLE
Anyone having issues lately with viewing frigate recordings on homeassistant?

### DISCUSSION THREAD

**Discussion by aaronjrodrigues** [blakeblackshear/frigate#15343] [Ask A Question] [answered]
I've noticed that for the past few weeks I've not been able to view my frigate media on homeassistant. I can view the live streams but just can't see the recorded media. I can see it perfectly fine when using the frigate ui.

**Comment by NickM-27** [comment, marked as answer]
this is a bug in HA 2024.11, fixed in HA 2024.12
